feature/interface/singleton/ate_tests/singleton_test/singleton_test.go

```diff
@@ -263,19 +266,20 @@ func (tc *testCase) verifyInterfaceDUT(
 			}
 		}
 	}
+if !*deviations.MissingNeighborOrigin {
```

Comment:
Still not addressed: https://github.com/openconfig/featureprofiles/pull/653#discussion_r1043675090